deurdu/server/Controllers/blogControllers.js

```diff
@@ -0,0 +1,187 @@
+import express from 'express';
+import mysql from 'mysql';
+import multer from 'multer';
+
+const router = express.Router();
+
+// MySQL database connection
+const db = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '',
+  database: 'deurdu',
+});
```

Comment:
this does not look correct to me, why should the connection be created with `root` user and without any password? please check how a user on Linux can created the connection with appropriate credentials.

Reply:
at least setting a password for `root` user seems possible
![mySqlConnectWithpassword](https://github.com/user-attachments/assets/9025b79e-3942-447d-a91c-fb42340e2a6d)